.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
       - "release/**"
   pull_request:
 env:
-  DOCKER_COMPOSE_VERSION: 1.29.2
+  DOCKER_COMPOSE_VERSION: 1.28.0
```

Comment:
How about a comment here and in `check-minimum-requirements.sh` that we want to keep these two values in sync?

https://github.com/getsentry/onpremise/blob/47c4a1df299fffb8340160d9633c05a6c8f75adc/install/check-minimum-requirements.sh#L4

Reply:
Sure, I'm fine to DRY up if you want.